yarn-project/simulator/src/avm/avm_simulator.test.ts

```diff
@@ -1084,7 +1081,8 @@ describe('AVM simulator: transpiled Noir contracts', () => {
           // infinitely loop back to the tested instruction
           // infinite loop should break on side effect overrun error,
           // but otherwise will run out of gas
-          new Jump(/*jumpOffset*/ 2),
+          // Note: 15 is the byte index, calculated as 3*size(Set.wireFormat8)
+          new Jump(/*jumpOffset*/ 15),
         ]);
```

Comment:
we should probably just use size of set here instead of having a magic 15 (even with the comment)?